src/tests/ftest/container/fill_destroy_loop.yaml

```
@@ -0,0 +1,17 @@
+hosts:
+  test_servers: 3
+server_config:
+  name: daos_server
+timeout: 500
+pool:
+  name: daos_server
+  control_method: dmg
+  scm_size: 1G
+  testloop: 100
+container:
+  control_method: daos
+  properties: "rf:1"
+  obj_class: OC_RP_2G1
+  block_size: 1024000
```

This should be data_size:
```suggestion
  data_size: 1024000
```